Handle case when a project doesn't have Lombok in any code

Assume you use the [delombok-action](https://github.com/sleberknight/delombok-action) on a project that doesn't have any Lombok in it. The output will be:

```
No files contain Lombok, so nothing to do. Exit with success.
```

And it will not create the output directory that would normally contain the Delomboked sources. By default the output directory is `src-delombok`.

If you then apply this action, it will currently fail with the error:

```bash
find: ‘src-delombok’: No such file or directory
Error: Process completed with exit code 1.
```
And because `find` returns with exit code 1, the overall action fails and GitHub puts a big red ❌ on the workflow.

Instead, this action, should handle the case when the input directory does not exist and print out something nicer, so that the workflow gets a nice big ✔️ instead.

--- a/action.yml
+++ b/action.yml
@@ -14,6 +14,6 @@ runs:
         echo "Inputs:"
         echo "  directory: ${{ inputs.delombokSourcePath }}"
         echo "---- Start of Delomboked source files ----"
-        find "${{ inputs.delombokSourcePath }}" -type f -print -exec cat -n {} \;
+        if [ -d "${{ inputs.delombokSourcePath }}" ]; then find "${{ inputs.delombokSourcePath }}" -type f -print -exec cat -n {} \; ; echo "No delomboked sources found. Directory ${{ inputs.delombokSourcePath }} does not exist" ; fi
         echo "---- End of Delomboked source files ----"
       shell: bash

--- a/action.yml
+++ b/action.yml
@@ -14,6 +14,6 @@ runs:
         echo "Inputs:"
         echo "  directory: ${{ inputs.delombokSourcePath }}"
         echo "---- Start of Delomboked source files ----"
-        if [ -d "${{ inputs.delombokSourcePath }}" ]; then find "${{ inputs.delombokSourcePath }}" -type f -print -exec cat -n {} \; ; else echo "No delomboked sources found. Directory ${{ inputs.delombokSourcePath }} does not exist." ; fi
+        if [ -d "${{ inputs.delombokSourcePath }}" ]; then find "${{ inputs.delombokSourcePath }}" -type f -print -exec cat -n {} \; ; else echo "No delomboked sources found. Directory '${{ inputs.delombokSourcePath }}' does not exist." ; fi
         echo "---- End of Delomboked source files ----"
       shell: bash

--- a/action.yml
+++ b/action.yml
@@ -14,6 +14,6 @@ runs:
         echo "Inputs:"
         echo "  directory: ${{ inputs.delombokSourcePath }}"
         echo "---- Start of Delomboked source files ----"
-        if [ -d "${{ inputs.delombokSourcePath }}" ]; then find "${{ inputs.delombokSourcePath }}" -type f -print -exec cat -n {} \; ; echo "No delomboked sources found. Directory ${{ inputs.delombokSourcePath }} does not exist" ; fi
+        if [ -d "${{ inputs.delombokSourcePath }}" ]; then find "${{ inputs.delombokSourcePath }}" -type f -print -exec cat -n {} \; ; else echo "No delomboked sources found. Directory ${{ inputs.delombokSourcePath }} does not exist" ; fi
         echo "---- End of Delomboked source files ----"
       shell: bash

--- a/action.yml
+++ b/action.yml
@@ -14,6 +14,6 @@ runs:
         echo "Inputs:"
         echo "  directory: ${{ inputs.delombokSourcePath }}"
         echo "---- Start of Delomboked source files ----"
-        if [ -d "${{ inputs.delombokSourcePath }}" ]; then find "${{ inputs.delombokSourcePath }}" -type f -print -exec cat -n {} \; ; else echo "No delomboked sources found. Directory ${{ inputs.delombokSourcePath }} does not exist" ; fi
+        if [ -d "${{ inputs.delombokSourcePath }}" ]; then find "${{ inputs.delombokSourcePath }}" -type f -print -exec cat -n {} \; ; else echo "No delomboked sources found. Directory ${{ inputs.delombokSourcePath }} does not exist." ; fi
         echo "---- End of Delomboked source files ----"
       shell: bash

